.github/workflows/nightly-link-checker.yml

```
@@ -0,0 +1,62 @@
+name: Nightly Job
+
+# on:
+#  schedule:
+#    - cron: "1 5 * * *" # This schedule runs every night at 12:01 AM Eastern time (adjust according to your timezone)
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      NODE_OPTIONS: '--max_old_space_size=3584'
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Checkout repository!
+        uses: actions/checkout@v3
+      - name: Setup node - ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+         node-version: ${{ matrix.node-version }}
+         cache: 'npm'
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Run script and capture logs  #broken link checker script
+        run: yarn build
+
+      - name: Run link checker #broken link checker script
+        run: yarn broken-link-checker:external > output.txt 2>&1 || true
+
+     # - name: Extract URLs from log
+        # run: |
+       #   cat output.txt | grep -Eo 'https?://[^ ]+' > urls.txt
+       # continue-on-error: true
+
+      - name: Extract URLs from log
+        run: |
+          cat output.txt | grep -oP 'Link: \Khttp[^\s]+' > urls.txt
+        continue-on-error: true
+
+      - name: Remove False Positives
+        run: |
+          comm -23 <(sort extracted_urls.txt) <(sort falsepos.txt) > filtered_urls.txt
```

```suggestion
          comm -23 <(sort urls.txt) < (sort falsepos.txt) > filtered_urls.txt
```
I'm not sure about extra space I've suggested. Need to debug it locally. Looking at the current error in GH action it complains about `urls.txtsort` file name